### ISSUE TITLE
No wsdl and customHeader

### ISSUE THREAD

**Issue by ChrisSFR** [artisaninweb/laravel-soap#75]
When I create a service and run the call method on my soapWrapper, then my client  wsdl and customHeader have no values (null)
My example Code:

```
$soapHeader = WsSecurity::createWsSecuritySoapHeader('foo', 'pwd', false);
		$this->soapWrapper->add('Auth', function ($service) use ($soapHeader) {
			$service
				->wsdl('https://my.exaple.com/foo?wsdl')
				->trace(true)
				->customHeader($soapHeader);
		});
		
		dd($this->soapWrapper->client('Auth', function ($client) {
			return $client;
		}));

```
Can someone reproduce this? Or am I the only one.

**Comment by artisaninweb** [comment]
You can call functions on your Soap service by:
```php 
$response = $this->soapWrapper->call('Auth.{function_name}');
```

**Comment by ChrisSFR** [comment]
Yeah I did:
`$response = $this->soapWrapper->call('Auth.getProductList', $data);`
My problem is, this call fails, because my client has no wsdl and customHeader set.
When I dump my `$this->soapWrapper` after calling `$this->soapWrapper->add...` there is a service Auth in my list, but no client is set. And the SoapWrapper client method tries to build the client new:

```
if (is_null($service->getClient())) {
        $client = new Client($service->getWsdl(), $service->getOptions()); // <---------
      } else {
        $client = $service->getClient();
      }
```

But no headers[] are passed to the client. And in my case no wsdl is set in the client constructor.

So the SoapWrapper->call fails when he gets the client.
But I don't know how to fix this.

**Comment by artisaninweb** [comment]
Can you try this:

```php
$this->soapWrapper->add('Auth', function ($service) use ($soapHeader) {
    $service
        ->wsdl('https://my.exaple.com/foo?wsdl')
        ->trace(true)
        ->client(null) <-- add this
        ->customHeader($soapHeader);
});
```

**Comment by artisaninweb** [comment]
Think i fixed the bug in version 0.3.0.1.
Please try it, thanks!
